sites/geohub/src/components/pages/map/Map.svelte

```diff
@@ -70,6 +70,8 @@
 		attributionControl: false
 	};
 
+	let showProgress = false;
```

Comment:
How about using ContextAPI and Stores for state of progress?

probably set context at +layout.svelte and getContext at Map.svelte, or any sub components want to use showProgress variable.